.gitbook/contents/hc/2_addsub/1_write-handler.md

```diff
@@ -6,15 +6,23 @@ The first step is to write our *off-chain handler*. This handler is responsible
 
 Let's begin by allowing our handler to receive two numbers. It will perform both addition and subtraction on these numbers. If the result of the subtraction results in an underflow (i.e. if `b > a` in the expression `a-b`), the handler will respond with an underflow error. Create a `Python` file in which to copy all the following code blocks.
 
-First, we'll need to import some `Python` libraries to help with our example:
+First, we'll need to import the Hybrid Compute SDK and some `Python` libraries to help with our example:
 
 ```python
+import { HybridComputeSDK } from '@bobanetwork/aa-hc-sdk-server';
```

Comment:
oh I see where the confusion is coming from now! In the docs you mixed now Python with TypeScript. 

The SDK is available in two different programming languages. 

In Python and in TypeScript. 

That's the python one: 
https://github.com/bobanetwork/aa-hc-sdk/tree/main/server-python

That's the typescript one: 
https://github.com/bobanetwork/aa-hc-sdk/tree/main/server

We should document both.